plugins/dependencies/src/test/resources/dependencies-versions-key-validated.txt

```diff
@@ -688,6 +712,14 @@ io.pivotal.spring.cloud..spring-cloud-services-starter-config-client=version.NO-
 io.pivotal.spring.cloud..spring-cloud-services-starter-service-registry=version.NO-RULE
 io.projectreactor..reactor-test=version.NO-RULE
 io.projectreactor.kotlin..reactor-kotlin-extensions=version.NO-RULE
+io.reactivex..rxjava=version.rxjava
+io.reactivex..rxkotlin=version.rxkotlin
+io.reactivex.rxjava2..rxandroid=version.rxandroid
+io.reactivex.rxjava2..rxjava=version.rxjava
+io.reactivex.rxjava2..rxkotlin=version.rxkotlin
+io.reactivex.rxjava3..rxandroid=version.rxandroid
+io.reactivex.rxjava3..rxjava=version.rxjava
+io.reactivex.rxjava3..rxkotlin=version.rxkotlin
```

Comment:
As you can see, we have a problem here: dependencies that don't have their version synchronized have the same version key. Related to one of my other comments above.

Reply:
Right. I think I actually did that on purpose with the idea that it would be simpler to move from RxJava2 to RxJava3 since the dependency name would be the same? But yea, I'll change it since I didn't realize people might want to import both at the same time.